src/components/Preferences/BodyKubernetes.vue

```diff
@@ -72,7 +74,10 @@ export default Vue.extend({
     },
     castToNumber(val: string): number | null {
       return val ? Number(val) : null;
-    }
+    },
+    kubernetesVersionLabel(): string {
```

Comment:
This works better as a computed property (since it's side-effect free).

Reply:
It didn't work the first time I tried it as one, but I'll retry (this is a recurring theme).
So after moving it to a computed method:
```
_vm.kubernetesVersionLabel is not a function
```

So the reference's parens need to be dropped. Yeah, this makes sense -- computed properties can never take arguments (because then you couldn't cache their values), so the references can never have parens.